sftp.go

```diff
@@ -239,7 +239,7 @@ func getSupportedExtensionByName(extensionName string) (sshExtensionPair, error)
 			return supportedExtension, nil
 		}
 	}
-	return sshExtensionPair{}, fmt.Errorf("Unsupported extension: %v", extensionName)
+	return sshExtensionPair{}, fmt.Errorf("unsupported extension: %s", extensionName)
```

Comment:
This is a fmt.Errorf, while the rest of the file uses errors.Errorf.

Reply:
Good point. Mostly, I didn’t want to disturb things too far, but this is probably a good candidate for switching.